ui-tests/src/test/java/org/openmrs/reference/page/HeaderPage.java

```diff
@@ -27,8 +27,8 @@ public void clickOnHomeLink() {
     }
 
     public void logOut() throws InterruptedException {	
-	 WebDriverWait waiter = new WebDriverWait(driver, 30);
-	 waiter.until(ExpectedConditions.visibilityOfElementLocated(By.className("logout")));
+	WebDriverWait waiter = new WebDriverWait(driver, 30);
```

Comment:
Any reason for instantiating `WebDriverWait` object here

Reply:
This being an Explicit wait within this class we needed to instantiate a webDriverWait  instance and assign it to the wait measured time , we can as well use the measured time by default that is being given in page class which also work

Reply:
I would rather recommend using a unique attribute for selenium to retrieve the element than instantiating the `WebDriverWait` object

Reply:
How can you retrieve an element at the same time instatiating an object of the webdriver, these are two different things

Reply:
this is a format or space/indention removal that should be reverted

Reply:
i resolved this by collecting the indentation, unless am mistakenly understanding you